### ISSUE TITLE
Add async book link in the Learn section?

### ISSUE THREAD

**Issue by Maxgy** [rust-lang/www.rust-lang.org#947]
Now that async await syntax has hit stable Rust, it might be a good idea to link the Rust Async book with the other books in the Learn section of the website.

**Comment by steveklabnik** [comment]
Maybe, it’s not done yet.

I’d also like to move it into the distribution, like the other books, when it’s ready.

**Comment by arijit79** [comment]
The async book contains many section which are still marked as todo. We should probably wait until it's completely finished

**Comment by qtfkwk** [comment]
Bump. Is it done enough yet?

Sections currently listed as "TODO" are: 6.4, 6.5, 10, and 11.

The most recent example of someone asking about async Rust is [this thread on Reddit](https://www.reddit.com/r/rust/comments/1egnf4p/comment/lftji0m/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) but there are others.

**Comment by senekor** [comment]
See https://github.com/rust-lang/async-book/issues/164

I don't think we should link the async book before its creators consider it done.

There are people actively working on adding async content to _the book_, which should be a little more high-level and targeted to users of async libraries.

**Comment by qtfkwk** [comment]
Agree. For clarification, this issue is about linking the async-book on the site, not putting async-book content in the rust book.
